docs/downstream-testing.md

```diff
@@ -34,11 +34,11 @@ As of May 2023, some projects' pipeline config files live under [the "pipeline"
 
 ## Testing Local Changes With All Downstream Projects
 
-To initiate a build for your local change, you'll need "Build & Read" access to https://buildkite.com/bazel/bazel-at-head-plus-downstream. If you don't have access to the pipeline, you can request it by filing an issue against https://github.com/bazelbuild/continuous-integration.
+To initiate a build for your local change, you'll need "Build & Read" access to https://buildkite.com/bazel/bazel-at-head-plus-downstream. If you are a core Bazel contributor, you can request the access by filing an issue against https://github.com/bazelbuild/continuous-integration, otherwise please reach out to someone who can have access to initate the downstream testing for your PR (e.g. `@meteorcloudy`, `@fweikert`, or `@Wyverald`).
```

Comment:
nit -- no need to have backticks around github usernames; they render as links anyway

Reply:
Done